ci-operator/step-registry/gcp/bastion/pre/gcp-bastion-pre-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: gcp-bastion-pre
+  from_image:
+    namespace: ocp
+    name: "4.5"
```

Comment:
Same - use updated installer to avoid installation bugs we didn't backport fixes to